src/Catalyst.Core.Modules.Ledger/Ledger.cs

```diff
@@ -136,30 +155,70 @@ public void Update(Cid deltaHash)
 
         private void UpdateLedgerFromDelta(Cid deltaHash)
         {
-            if (!_synchroniser.DeltaCache.TryGetOrAddConfirmedDelta(deltaHash, out var nextDeltaInChain))
+            int stateSnapshot = _stateDb.TakeSnapshot();
+            int codeSnapshot = _codeDb.TakeSnapshot();
+            if (stateSnapshot != -1 || codeSnapshot != -1)
             {
-                _logger.Warning(
-                    "Failed to retrieve Delta with hash {hash} from the Dfs, ledger has not been updated.", deltaHash);
-                return;
+                if (_logger.IsEnabled(LogEventLevel.Error))
+                    _logger.Error("Uncommitted state ({stateSnapshot}, {codeSnapshot}) when processing from a branch root {branchStateRoot} starting with delta {deltaHash}", stateSnapshot, codeSnapshot, null, deltaHash);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 220 characters long line (which is greater than 200 authorized).](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)